lib/services/apiAppManagement/package.json

```diff
@@ -17,7 +17,7 @@
     "management",
     "apiapp"
   ],
-  "main": "./lib/apiapp.js",
+  "main": "./lib/apiApp.js",
```

Comment:
yesterday, we just renamed the physical file name in the linux box to see it worked, but updating the entry in the  package.json is also good. merging